Makefile

```diff
@@ -66,6 +66,10 @@ update-gofmt:
 	hack/make-rules/update/gofmt.sh
 verify-gofmt:
 	hack/make-rules/verify/gofmt.sh
+update-labels:
+	hack/make-rules/update/labels.sh
+verify-labels:
+	hack/make-rules/verify/labels.sh
```

Comment:
These should also be `.PHONY`.
https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html

Reply:
added